### PR TITLE
Change to return FrameworkElement instead of UIElement

### PR DESCRIPTION
**Pull request by RebeccaAnne**
Fix for #563

### CODE REVIEW

File: source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl [hunk 1]
```diff
@@ -452,7 +452,6 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
     {
         [propget] HRESULT AdaptiveCard([out, retval] AdaptiveCard** value);
         [propget] HRESULT Errors([out, retval] Windows.Foundation.Collections.IVector<IAdaptiveError*>** value);
-        [propget] HRESULT Warnings([out, retval] Windows.Foundation.Collections.IVector<IAdaptiveWarning*>** value);
     };
```
Comment (khouzam) [inline source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl]: Seems unrelated. Also, should this stay in the interface even if we don't populate it yet?

Comment (RebeccaAnne) [reply source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl]: I'll move this to a separate change.

Per the discussion in the meeting the other day, I thought we had decided that we didn't want to have this. I'll double check with Matt before making this change. In any case, we can always add it back later if we change our mind.

---
In reply to: [171363004](https://github.com/Microsoft/AdaptiveCards/pull/1183#discussion_r171363004) [](ancestors = 171363004)
